plugins/jobs/web_client/main.js

```diff
@@ -1,3 +1,4 @@
+import $ from 'jquery';
```

Comment:
Yes it could go away I think, it was mostly to be able to access it from the console...